server/routes.js

```diff
@@ -72,7 +72,7 @@ module.exports = (app) => {
    * When we refactor PublicGroup to fetch the group in the container, we can remove
    * the explicit routes and just do `app.use(render)`
    */
-  app.get('/', mw.ga, mw.addTitle('OpenCollective - Collect and disburse money transparently'), render);
+  app.get('/', mw.ga, controllers.homepage, mw.addTitle('OpenCollective - Collect and disburse money transparently'), render);
```

Comment:
Middlewares, as their name suggests, should be in the middle. You should end with the controller.